card/mod/core/set/all/actify.rb

```diff
@@ -128,3 +128,11 @@ def with_transaction_returning_status
 def success
   Env.success(cardname)
 end
+
+def name_before_act
+  name_before_last_save || name_was
+end
```

Comment:
I was thinking if the names are not obvious enough you can look it up in the rails documentation, but I'll write a comment for you.